modules/serverless-ossm-setup-without-kourier-mtls.adoc

```diff
@@ -0,0 +1,199 @@
+[id="serverless-ossm-setup-without-kourier-mtls_{context}"]
+= Integrating {ProductShortName} and {ServerlessProductName} with mTLS enabled
+
+.Procedure
+
+. Create a `ServiceMeshControlPlane` object in the `istio-system` namespace that has mTLS enabled.
+. Add the namespaces that you would like to integrate with {ProductShortName} to the `ServiceMeshMemberRoll` object as members:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+  namespace: istio-system
+spec:
+  members: <1>
+    - knative-serving
+    - <namespace>
+----
+<1> A list of namespaces to be integrated with {ProductShortName}.
++
+[IMPORTANT]
+====
+This list of namespaces must include the `knative-serving` namespace.
+====
+. Apply the `ServiceMeshMemberRoll` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+. Create the necessary gateways so that {ProductShortName} accepts traffic:
++
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: knative-ingress-gateway
+  namespace: knative-serving
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - port:
+        number: 443
+        name: https
+        protocol: HTTPS
+      hosts:
+        - "*"
+      tls:
+        mode: SIMPLE
+        credentialName: <wildcard_certs> <1>
```

Comment:
```suggestion
        credentialName: <wildcard-certs> <1>
```
Better align with example openssl command above.

Reply:
It's part of our OCP docs guidelines to use `_` for variables